src/server/verification/verificationAPI.js

```diff
@@ -139,7 +139,7 @@ const setup = (app: Router, verifier: VerificationAPI, gunPublic: StorageAPI, st
         // if user is already verified, we're skipping enroillment logic
         if (user.isVerified || disableFaceVerification || allowDuplicatedFaceRecords || isE2ERunning) {
           // creating enrollment session manually for this user
-          const enrollmentSession = enrollmentProcessor.createEnrollmentSession(user, log)
+          const enrollmentSession = enrollmentProcessor.createEnrollmentSession(user, enrollmentIdentifier, log)
```

Comment:
there's probably not reason to add enrollmentIdentifier since it is probably accessible via the next code line
initialize(payload)

Reply:
it is not in the payload it is in the params